sources/HUBComponentModelBuilderImplementation.m

```diff
@@ -532,7 +562,21 @@ - (HUBComponentImageDataBuilderImplementation *)getOrCreateBuilderForCustomImage
     return newBuilder;
 }
 
-- (HUBComponentModelBuilderImplementation *)getOrCreateBuilderForChildWithIdentifier:(nullable NSString *)identifier
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdirect-ivar-access"
```

Comment:
Move this into the method instead, just around where you need it, so it's clear why you are doing this.